core/bc_contract.go

```diff
@@ -145,7 +145,7 @@ func (bc *BaseContract) QueryGetNonce(owner *types.Address) (string, error) {
 			// let's just say it's an old nonsense
 			lastNonce.Nonce = []uint64{new(big.Int).SetBytes(data).Uint64()}
 		}
-		exist = strconv.FormatUint(lastNonce.GetNonce()[len(lastNonce.GetNonce())-1], 10)
+		exist = strconv.FormatUint(lastNonce.GetNonce()[len(lastNonce.GetNonce())-1]/1e6, 10)
```

Comment:
To improve clarity, please define a constant for the value 1e6. And please write a comment for the line.

Reply:
fix